substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/jfr/remote/JfrAutoSessionManager.java

```diff
@@ -0,0 +1,152 @@
+package com.oracle.svm.core.jdk.jfr.remote;
```

Comment:
Please add a license header for this file.

Reply:
done.